Event Subscription Get Started update (#13004)
kyma-project/kyma: docs/02-get-started/04-trigger-workload-with-event.md

## Create a Subscription

Next, to subscribe to an event so that we can actually listen for it, we need a [Subscription](../05-technical-reference/00-custom-resources/evnt-01-subscription.md) and an [Application](https://kyma-project.io/docs/kyma/latest/05-technical-reference/00-custom-resources/ac-01-application/#documentation-content) custom resources. We're going to be listening for an event of type `order.received.v1`. 

<div tabs name="Create a Subscription" group="trigger-workload">

>   <details open>

Reviewer: One thing that's interesting is that the kubectl version works even without the application created, so I wonder if the **Application** field should filled in by selecting a value from the drop-down list, or whether it should rather be a text field to fill in (I assume it would work, as you also provide the name of the app in the **eventType.value** (line 153) in the Subscription CR via kubectl and it just works. 

Can you take it up with your team?

Reviewer: As discussed via DM, an option to either type in an Application or select it from the drop-down list will be added (i.e. both will be possible), which solves the issue. Thanks!

  Kyma Dashboard
  </summary>

1. Go back to your `lastorder` Function's view in Kyma Dashboard.
2. In the **Configuration** tab, click on **Create Subscription+**.
3. Provide the following parameters:
    - **Subscription name**: `lastorder-sub`

>     - **Application name**: `myapp`

Reviewer: ```suggestion
    - **Application name**: `myapp` (use the **Advanced** view for this)
```
I suggest we add one more change here for now

Reviewer: Or not... :D

  Kyma Dashboard
  </summary>

> 1. Using the left navigation go to **Apllications**.

Reviewer: ```suggestion
1. Using the left navigation, go back to the main view > **Applications**.
```

2. Click on **Create Application+**.

> 3. Provide a name for your application, for example `myapp`.

Reviewer: ```suggestion
3. Provide a name for your application, for example `myapp`, and click **Create**.
```

4. Go to **Namespaces** and click on `default`.
5. Using the left navigation, go back to **Workloads** > **Functions**.
6. Select your `lastorder` Function and navigate to the **Configuration** tab.
7. Click on **Create Subscription+**.
8. Provide the following parameters:

>     - **Subscription name**, for example `received-orders`

Reviewer: ```suggestion
    - **Subscription name**: `lastorder-sub`
```
Note, that we also have steps for kubectl, so we should aim to keep them (including the values used) as consistent as possible, just in case someone prefers to perform some steps via kubectl and others via Kyma Dashboard.

>     - **Application name**: choose `myapp`

Reviewer: ```suggestion
    - **Application name**: `myapp`
```

    - **Event name**: `order.received`

>     - **Event version**: `v1`
> 
>     **Event type** is generated automatically; in this example: `myapp.order.received.v1`. 

Reviewer: ```suggestion
    - **Event version**: `v1`

    - **Event type** is generated automatically. For this example, it's `sap.kyma.custom.myapp.order.received.v1`. 
```

1) The **Event type** has changed, too: 
![image](https://user-images.githubusercontent.com/53601578/148782655-b0770c7d-1ccc-4e0a-865a-fba94ab36cf4.png)

2) This is currently rendered incorrectly on the [Website](https://deploy-preview-13004--kyma-project-docs-preview.netlify.app/docs/kyma/preview/02-get-started/04-trigger-workload-with-event/), that's why I added it as another list element. It fits, as it refers to one of the required values. 
![image](https://user-images.githubusercontent.com/53601578/148782178-2aec4699-024e-4873-97f8-ae423402d139.png)

    
9. Click **Create**. 

>    The name of the event Subscription is generated automatically.

Reviewer: ```suggestion
```

It's not any more. You give the name, and it's required. 
![image](https://user-images.githubusercontent.com/53601578/148782772-73d6ac47-999f-40c9-a4d8-e1483e0601d6.png)

> 1. Using the left navigation, go back to **Workloads** > **Functions**.
> 2. Select your `lastorder` Function and navigate to the **Configuration** tab.

Reviewer: ```suggestion
1. Go back to your `lastorder` Function's view in Kyma Dashboard.
```
You never really navigate away from it Dashboard-wise, so it's a bit confusing to have these steps repeated here.

> 3. Click on **Create Subscription+**.
> 4. Provide the following parameters:

Reviewer: ```suggestion
2. In the **Configuration** tab, click on **Create Subscription+**.
3. Provide the following parameters:
```
Withe the change to the initial steps, the numbering must be updated.

    - **Subscription name**: `lastorder-sub`
    - **Application name**: `myapp`
    - **Event name**: `order.received`
    - **Event version**: `v1`

    - **Event type** is generated automatically. For this example, it's `sap.kyma.custom.myapp.order.received.v1`. 
    

> 5. Click **Create**. 

Reviewer: ```suggestion
4. Click **Create**. 
```
Withe the change to the initial steps, the numbering must be updated.
